package.json

```diff
@@ -15,6 +15,8 @@
     "relay": "^0.8.0-1"
   },
   "scripts": {
+    "prepare": "husky install",
```

Comment:
I _think_ this probably copies what's configured in the package.json into a pure git hook? Not totally sure, but it's [in the setup docs](https://typicode.github.io/husky/#/?id=install).